src/scripts/start_celery_beat.sh

```diff
@@ -5,12 +5,8 @@
 # after a deployment or when the container is restarted.
 
 echo "Entering start celery event script"
-export DJANGO_SETTINGS_MODULE=$1
-if [ -z "$DJANGO_SETTINGS_MODULE" ]
-  then
-    echo "This script expects the DJANGO_SETTINGS_MODULE as first argument"
-fi
 echo "Using Django settings module: $DJANGO_SETTINGS_MODULE"
+CONCURRENCY="${CELERY_BEAT_CONCURRENCY:-4}"
 python manage.py compilemessages
-celery -A core worker --beat
+celery -A core worker --beat  --concurrency=$CONCURRENCY --loglevel info
```

Comment:
```suggestion
celery -A core worker --beat  --concurrency=${CELERY_BEAT_CONCURRENCY:=4} --loglevel info
```
pour éviter la variable intermédiaire CONCURRENCY ?

Reply:
Quelle élégance!
J'en ai profiter pour changer le nom "event" en "events".